### PR TITLE
Fix prompt for task progress

### PR DESCRIPTION
**Pull request by eed3si9n**
Some minor improvement/fixes for prompts + logging around super shell.
This should fix the behavior where the prompt overwrites the last output like for `show name` etc.

Ref #4461

### CODE REVIEW

File: main/src/main/scala/sbt/internal/TaskProgress.scala [hunk 1]
```diff
@@ -86,20 +86,24 @@ private[sbt] final class TaskProgress(currentRef: ProjectRef) extends ExecutePro
...
-    console.println("")
-    console.println("")
-    console.println("")
-    console.println("")
-    console.println("")
-    console.println("")
-    console.print(CursorUp5)
+    if (isLogReady.get) ()
+    else {
+      console.println("")
```
Comment (eatkins) [inline main/src/main/scala/sbt/internal/TaskProgress.scala]: This is a bit inscrutable. Maybe you could add a comment that explains why you need to print these blank lines followed by CursorUp5? I'd also find this slightly more readable if it were:
```scala
console.print(s"${"\n" * 5}$CursorUp5")
```

File: main/src/main/scala/sbt/internal/Aggregation.scala [hunk 2]
```diff
@@ -77,8 +77,6 @@ object Aggregation {
       if (show.taskValues) printSettings(r, show.print)
     }
     if (show.success) printSuccess(start, stop, extracted, success, log)
-    // wait for async logger to catch up
-    Thread.sleep(100)
```
Comment (eatkins) [inline main/src/main/scala/sbt/internal/Aggregation.scala:81]: Always great to remove a Thread.sleep.

File: main/src/main/scala/sbt/internal/TaskProgress.scala [hunk 3]
```diff
@@ -86,20 +86,24 @@ private[sbt] final class TaskProgress(currentRef: ProjectRef) extends ExecutePro
 
   private[this] val console = ConsoleOut.systemOut
   private[this] def readyLog(): Unit = {
```
Comment (eatkins) [inline main/src/main/scala/sbt/internal/TaskProgress.scala]: Could this itself just be a lazy val and then you could get rid of the AtomicBoolean?

**Review by eatkins — DISMISSED** [review]
I don't feel strongly about my suggestions so you should feel free to merge with or without them. I'm hoping these changes will make super shell more useable for me!

File: main/src/main/scala/sbt/internal/TaskProgress.scala (hunk 3 above)
Comment (eed3si9n) [reply main/src/main/scala/sbt/internal/TaskProgress.scala]: That's a good idea.

File: main/src/main/scala/sbt/internal/TaskProgress.scala (hunk 1 above)
Comment (eed3si9n) [reply main/src/main/scala/sbt/internal/TaskProgress.scala]: I initially added this to make space on the bottom for task progress. I am going to see if we still need this actually.

**Pull request merged**